packages/oracle-manager/contracts/modules/core/OracleManagerModule.sol

```diff
@@ -0,0 +1,120 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IOracleManagerModule.sol";
+import "../../utils/ReducerNodeLibrary.sol";
+import "../../utils/ExternalNodeLibrary.sol";
+import "../../utils/PythNodeLibrary.sol";
+import "../../utils/ChainlinkNodeLibrary.sol";
+
+import "../../storage/Node.sol";
+import "../../storage/NodeDefinition.sol";
+
+contract OracleManagerModule is IOracleManagerModule {
+    error UnsupportedNodeType(uint nodeType);
+    error NodeNotRegistered(bytes32 nodeId);
```

Comment:
move these to the interface?  seems like that's the pattern i've noticed.  nit pick :)

Reply:
i dont think we have been moving the errors to the interface, I'm not sure why though.